plugins/by-name/cmp-tabby/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib, ... }:
+let
+  inherit (lib) types;
+  inherit (lib.nixvim) defaultNullOpts;
+in
+lib.nixvim.neovim-plugin.mkNeovimPlugin {
+  name = "cmp-tabby";
+
+  maintainers = [ lib.maintainers.GaetanLepage ];
+
+  imports = [
+    { cmpSourcePlugins.cmp_tabby = "cmp-tabby"; }
```

Comment:
IDK why, I need to double check this every time...

It is correct though :+1: 

https://github.com/nix-community/nixvim/blob/5abe382c54eff3cd3977733e9d268ab8994f397b/plugins/cmp/auto-enable.nix#L38